Calibration/IsolatedParticles/test/proto_runIsolatedGenParticles_cfg.py

```diff
@@ -35,12 +35,12 @@
                                            maxd0            = cms.double(10.0)
                                            )
 
-process.load("Calibration.IsolatedParticles.isolatedGenParticles_cfi")
-process.isolatedGenParticles.GenSrc            = "Dummy"
-process.isolatedGenParticles.UseHepMC          = False
-#process.isolatedGenParticles.PTMin             = 0.0
-process.isolatedGenParticles.Verbosity         = 0
-#process.isolatedGenParticles.MaxChargedHadronEta = 3.5
+process.load("Calibration.IsolatedParticles.IsolatedGenParticles_cfi")
```

Comment:
I might be wrong, but I believe the CMS naming convention is to start with lower case.

Reply:
This is the name of the file created in cfipython

Reply:
well, then you should adapt `fillDescriptions` accordingly